campaign-launcher/server/src/modules/campaign/campaign.service.ts

```diff
@@ -125,7 +125,7 @@ export class CampaignService {
 
     this.logger.log(`Setting up escrow with manifest`);
     const escrowConfig = {
```

Comment:
This function shouldn't even exist, the campaigns are launched interacting from the frontend directly with the blockchain.
Also the endpoint that calls this, should not exist. This change should have been done in the frontend.